### PR TITLE
added setup.cfg

### PR DESCRIPTION
**Pull request by domdinicola**
maybe we can include .flake8 content here as well

### CODE REVIEW

**Comment by caktus-philip** [comment]
@domdinicola I think flake8 settings have to be in the EquiTrack directory otherwise they won't be copied into Docker. See #736. @vkurup might be able to comment more.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+include_trailing_comma = true
+known_first_party = EquiTrack, assets, attachments, audit, environment, firms, funds, locations, management, monitoring, notification, partners, publics, reports, supplies, t2f, tpm, users, utils, vision, workplan
+multi_line_output = 0
+line_length = 80
+balanced_wrapping = true
+order_by_type = false
+not_skip = __init__.py
+
+[pep8]
+ignore = E501,E401,W391,E128,E261,E731
```
Comment (vkurup) [inline setup.cfg]: Personally, I wouldn't be in favor of ignoring these. Note that we just spent a bit of time cleaning up the entire repo of all flake8 violations, so allowing them back in would feel a little like a step backwards. If we do agree to change this policy, we should document it here: https://github.com/unicef/etools/blob/master/docs/guidelines.rst#flake8-exceptions

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+max-line-length = 160
```
Comment (vkurup) [inline setup.cfg]: My main argument for keeping it at 120 or lower is that longer lines often go past the right edge of the Github display box, so tend not to get reviewed as well.

**Comment by vkurup** [comment]
Never heard of `isort` before, btw. Sounds interesting!

**Comment by reinbach** [comment]
Just to let everyone know my preference, cause I know you're all dying to know what it is;

The closer we are to `pep8` standard the happier I am. So the less `ignore`s we have the better.
I'm old school, I still prefer line lengths being 80, but I know I'm not going to get any other takers on that one :)

**Comment by domdinicola** [comment]
Looks like isort follows (or can be configured to follow) what Organizing imports section of the guideline says.

File: setup.cfg (hunk 1 above)
Comment (domdinicola) [reply setup.cfg]: I was not aware of the doc!
Looks really useful in order to be aligned.

**Comment by caktus-philip** [comment]
@domdinicola Thanks for `isort`. Sorting imports statements is something I do begrudgingly if at all. I'd be glad not to have to do it anymore. I think organizing imports in the Python/django/3rd party/projects sections provides more benefit, so if it can do that too, that's great.

**Comment by domdinicola** [comment]
@caktus-philip latest commit should do what you were asking for

**Pull request merged**